ops/storage-ddls/cassandra-setup.cql

```diff
@@ -79,87 +76,126 @@ CREATE TYPE features (
 );
 
 CREATE TABLE computedtiles (
+    tileid text,
     tilex int,
     tiley int,
     tilez int,
-    tileid text,
     periodstartdate timestamp,
     periodenddate timestamp,
     periodtype text,
     period text,
-    pipeline text,
-    sourceid text,
-    topic text,
-    topiclangcode text,
+    pipelinekey text,
+    externalsourceid text,
+    topicconjunctionid uuid,
     insertion_time timestamp,
     heatmap text,
     placeids frozen<set<text>>,
```

Comment:
In what scenarios would we read the placeids column? If memory serves me well, the field was originally here so that we could easily get the tile for a given wof-id (I don't think we should do this since secondary indices aren't very performant in Cassandra). If we're not going to index on this field and/or use it for retrievals, I wonder if we can just ditch the field.